scripts/bmm_examples/Example2_Loaiza&Souza2018.R

```diff
@@ -74,86 +75,116 @@ levels(wmData$ageGroup) <- c("Young","Old")
 levels(wmData$RI) <- c("short","long")
 levels(wmData$cueCond) <- c("No","Retro")
 
+# specfiy contrasts
+contrasts(wmData$ageGroup) <- bayestestR::contr.equalprior(2)
```

Comment:
I don't have this package installed so it gave me an error "Error in loadNamespace(x) : there is no package called ‘bayestestR’". Perhaps include it in the p_load function?

Reply:
I'm also curious what this does as I haven't used it before, so maybe you could explain it to me (or is it in the paper already?) :)

Reply:
I have now included the package in the `p_load` statement. These priors are designed so that each condition mean has the same prior width after effects have been added up to the Intercept. For designs with factors that only have two-levels, like in the example, this is not super critical and you could also use `contr.sum` but as soon as a factor has 3 or more levels these contrasts are quite useful to have an equal prior width for all condition means: https://easystats.github.io/bayestestR/reference/contr.equalprior.html